recipe/build.sh

```diff
@@ -11,11 +11,34 @@ if [ `uname` == Darwin ]; then
 fi
 
 if [ `uname` == Linux ]; then
-    if [ "$PY_VER" == "2.7" ]; then
-        pip install --no-deps https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.9.0-cp27-none-linux_x86_64.whl
-    elif [ "$PY_VER" == "3.4" ]; then
-        pip install --no-deps https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.9.0-cp34-cp34m-linux_x86_64.whl
-    elif [ "$PY_VER" == "3.5" ]; then
-        pip install --no-deps https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.9.0-cp35-cp35m-linux_x86_64.whl
-    fi
+
+    # Install Bazel from source
```

Comment:
I could not figure out what was wrong with bazel. If you want, please open a new pull in staged-recipes and iterate on bazel based on my recipe.